persistence/shared_sql.go

```diff
@@ -92,7 +92,7 @@ func (p *PostgresContext) getActor(actorSchema types.ProtocolActorSchema, addres
 	if err != nil {
 		return
 	}
-
+	actor.ActorType = actorSchema.GetActorType()
```

Comment:
`getActor` uses `getActorFromRow` under the hood

BUT

`GetAllXXX` uses `getActorFromRow` directly

So this won't actually fully work.

I'm okay with leaving it as is because we have a ton of scope creep, but could you add a TODO to fix it (in a follow up PR). I also think this is a really good opportunity to do some TDD to be future-proof.

Reply:
I coudn't resist... https://github.com/pokt-network/pocket/pull/403/commits/fb42810947fe706561077ad191ee6a81246dab43 and https://github.com/pokt-network/pocket/pull/403/commits/6ff3c3ad2b446443df150931103c1b2b60df562b